.github/workflows/ci.yml

```diff
@@ -131,6 +131,7 @@ jobs:
   coverage:
     continue-on-error: true
     runs-on: ubuntu-latest
+    if: ${{ ! contains(github.event.pull_request.labels.*.name, 'dependencies') }}
```

Comment:
should we also add the check for the author, in case someone tries to skip the coverage test just by adding label :)

```!contains(github.event.pull_request.user.login, 'dependabot')```

Reply:
On more thought, this is not necessary. Resolving it :)